website/content/en/docs/deployment/add-ons/load-balancer/guide.md

```diff
@@ -122,14 +127,15 @@ while ! kustomize build deployments/add-ons/load-balancer | kubectl apply -f -;
 ### Update the domain with ALB address
 
 1. Check if ALB is provisioned. This may take a few minutes.
-    1. ```
+    ```
         kubectl get ingress -n istio-system istio-ingress
         NAME            CLASS    HOSTS   ADDRESS                                                                  PORTS   AGE
         istio-ingress   <none>   *       xxxxxx-istiosystem-istio-2af2-1100502020.us-west-2.elb.amazonaws.com   80      15d
-        ```
-    2. If `ADDRESS` is empty after a few minutes, check the logs of controller by following [this guide](https://awslabs.github.io/kubeflow-manifests/docs/troubleshooting-aws/#alb-fails-to-provision)
-1. When ALB is ready, copy the DNS name of that load balancer and create a CNAME entry to it in Route53 under subdomain (`platform.example.com`) for `*.platform.example.com`
-    1. ![subdomain-*.platform-record](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/website/content/en/docs/images/load-balancer/subdomain-*.platform-record.png)
+    ```
+    If `ADDRESS` is empty after a few minutes, check the logs of the controller by following the troubleshooting steps in [ALB fails to provision](https://awslabs.github.io/kubeflow-manifests/docs/troubleshooting-aws/#alb-fails-to-provision).
+1. When ALB is ready, copy the DNS name of that load balancer and create a CNAME entry to it in Route53 under the subdomain (`platform.example.com`) for `*.platform.example.com`. Please note that it might make anywhere from five to ten minutes for DNS changes to propagate and for your URL to work.
```

Comment:
`might make anywhere from five to ten minutes`

can we use the language upto 5-10mins instead of a range 5-10 because sometimes it is pretty quick.

and for helping customer ask them to use this tool to check if the DNS entry propagated: https://toolbox.googleapps.com/apps/dig/#CNAME/